zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/FileSystemStorage.java

```diff
@@ -60,16 +62,39 @@
   public FileSystemStorage(ZeppelinConfiguration zConf, String path) throws IOException {
     this.zConf = zConf;
     this.hadoopConf = new Configuration();
+    URI zepConfigURI;
+    URI defaultFSURI;
+
+    zepConfigURI = getURI(path, "Failed to get URI path");
```

Comment:
It is not a good practice to pass error message to function. And is `getURI` necessary ? I see it only create new URI but do nothing else.

Reply:
I have pushed requested changes.

Reply:
@zjffdu can you please take a look and everything is ok then we can merge it.

Reply:
@zjffdu Did you get a chance to look at it?